package.json

```diff
@@ -12,8 +12,10 @@
     "standard": "echo 'standard is no longer being used, use `npm run eslint` instead!' && exit 1",
     "dev": "webpack --config webpack.dev.js --progress --colors --watch",
     "heroku-prebuild": "bin/heroku",
-    "build": "webpack --config webpack.prod.js --progress --colors --bail",
-    "start": "tsc && sequelize db:migrate && node built/app.js"
+    "build": "npm run-script build-backend && npm run-script build-frontend",
+    "build-backend": "tsc",
+    "build-frontend": "webpack --config webpack.prod.js --progress --colors --bail",
+    "start": "npm run-script build-backend && sequelize db:migrate && node built/app.js"
```

Comment:
Why does start compile both things? I would think it only starts my already compiled stuff.

Reply:
Mainly because we did it before. I agree it's reasonable to change this.